components/store/StoreCard.js

```diff
@@ -67,17 +76,17 @@ export default function StoreCard({ store, callBack, seeProduct }) {
         {seeProduct && <StoreProductButton callBack={callBack} />}
       </SpaceBetweenRowContainer>
       <Caption style={{ marginBottom: 4 }} color={Colors.secondaryText}>
-        {distance} miles away
+        {`${distance} miles away`}
```

Comment:
Not super related but now I'm thinking we might need to look into a different way to calculate distance. Currently, it's always unrealistically low bc it's calculated as the crow flies.

Reply:
Conversation in #68